### ISSUE TITLE
Can we implement random forest using xgboost dart?

### ISSUE THREAD

**Issue by haruji** [dmlc/xgboost#2028]
I wonder that if users can use xgboost to implement random forest, and I think it may be an attractive issue due to the efficiency of xgboost compared to sklearn. Seems that the new model dart with parameters eta=1, drop rate=1, normalize_type=forest can easily achieve the goal. Could you please tell me is that correct? Thanks!

**Comment by marugari** [comment]
See https://github.com/dmlc/xgboost/issues/2018

As for DART, `{'booster': 'gbtree', 'num_parallel_tree': N}` is equivalent to `{'booster': 'dart', 'drop_rate': 1.0}`.

**Comment by julioasotodv** [comment]
I believe that it's not true. `{'booster': 'dart', 'drop_rate': 1.0}` will give you just random guesses (in fact, all samples predicted always with the same class), rather than a random forest implementation.
